Makefile

```diff
@@ -87,12 +87,12 @@ else
 endif
 
 ifdef ROCM_PATH
-	CFLAGS += -I$(ROCM_PATH)/hip/include
-	LDFLAGS += -L$(ROCM_PATH)/hip/lib
+	CFLAGS += -I$(ROCM_PATH)/hip/include -I$(ROCM_PATH)/hsa/include
```

Comment:
Ya it is. It is easier to read this way. I'll change it if we face any big issue in the future.